bot/core/events/registry.py

```
@@ -0,0 +1,468 @@
+import importlib
+import uuid
+from abc import ABC, abstractmethod
+from dataclasses import dataclass, field
+from datetime import datetime
+from enum import Enum, auto
+from typing import Any, Dict, Optional, Type, TypeVar, Generic, Union, ClassVar
+
+from bot.utils.time import get_utc_now
+
+
+class EventType(Enum):
```

Ruff (0.8.2)

Redefinition of unused `EventType` from line 9

(F811)